Bottom: d9b25ff39ffd41152305fa1588d21dc9eefaec40
Top:    af9053d7d5a151d3cc54f3315ffdc06d0f0f6643
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:05:42 -0500

Refresh of tmp-mantis-database

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 08c65221..241372b3 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -123,7 +123,7 @@
 
 - name: start container (mysql)
   docker_container:
-    name: '{{ mantisbt_db_dv_name}}'
+    name: '{{ mantisbt_db_container_name}}'
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
     image: '{{ mantisbt_db_image_name }}'
@@ -141,7 +141,7 @@
 - name: wait for initialization to complete
   shell: >
       printf 'SHOW GLOBAL STATUS\n' | docker exec -i 
-      '{{ mantisbt_db_dv_name}}'
+      '{{ mantisbt_db_container_name}}'
       mysql --host=localhost 
       --user={{ mantisbt_db_user | quote }}
       --password={{ mantisbt_db_password | quote }}
